### PR TITLE
Make use of a better python module for gitlab_group

### PR DESCRIPTION
**Pull request by dj-wasabi**
##### SUMMARY
Using a python module that is not deprecated anymore.

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME
gitlab_group

##### ANSIBLE VERSION

```
ansible 2.5.2
  config file = None
  configured module search path = [u'/Users/wdijkerman/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/wdijkerman/git/ansible/environment/.virtualenv/environment/lib/python2.7/site-packages/ansible
  executable location = /Users/wdijkerman/git/ansible/environment/.virtualenv/environment/bin/ansible
  python version = 2.7.10 (default, Oct  6 2017, 22:29:07) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.31)]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/modules/source_control/gitlab_group.py [hunk 1]
```diff
@@ -5,23 +5,21 @@
...
 
-
 DOCUMENTATION = '''
 ---
 module: gitlab_group
 short_description: Creates/updates/deletes Gitlab Groups
 description:
    - When the group does not exist in Gitlab, it will be created.
-   - When the group does exists and state=absent, the group will be deleted.
+   - When the group does exist and state=absent, the group will be deleted.
```
Comment (Akasurde) [inline lib/ansible/modules/source_control/gitlab_group.py:18]: Add a note about Python library change like you did in gitlab_project PR. Rest LGTM

Comment (dj-wasabi) [reply lib/ansible/modules/source_control/gitlab_group.py:18]: Thanks! Added a line that I also have added in different PR for gitlab_project.

**Comment by Akasurde** [comment]
@resmo could you please review this ?

**Review by resmo — APPROVED** [review]
I will add a entry in the 2.7 migration docs.

LGTM

**Pull request merged**